packages/concerto-cto/lib/printer.js

```diff
@@ -386,9 +386,17 @@ function toCTO(metaModel) {
             case `${MetaModelNamespace}.ImportAllFrom`:
                 result += `\nimport ${imp.namespace}.*`;
                 break;
-            case `${MetaModelNamespace}.ImportTypes`:
-                result += `\nimport ${imp.namespace}.{${imp.types.join(',')}}`;
+            case `${MetaModelNamespace}.ImportTypes`:{
```

Comment:
Are there any tests for this change?

Reply:
Yes, Test cases are evaluated for both parser and printer. 

CTO(original) --[parser]--> AST(JSON) --[printer]--> CTO(new)

we check if CTO(original) == CTO(new)

So we are checking whether the printer is working correctly or not and it is true for all the test cases (where aliasing is not done )